Include/internal/pycore_object.h

```diff
@@ -10,6 +10,10 @@ extern "C" {
 
 #include "pycore_pystate.h"   /* _PyRuntime */
 
+/* Private API for the LOAD_METHOD opcode. */
```

Comment:
I think this comment is not needed.
We have detailed comment in function definition.